code/modules/client/preference_setup/loadout/loadout_ears.dm

```diff
@@ -13,6 +13,18 @@
 	display_name = "headphones"
 	path = /obj/item/clothing/ears/earmuffs/headphones
 
+/datum/gear/ears/double/earphones
```

Comment:
Not a requirement but i'm curious why this isn't combined with the headphones above instead?

Reply:
It could be, but I figured they were different enough to warrant their own entry instead of "audio device selection" or somesuch.